ambry-router/src/main/java/com.github.ambry.router/GetBlobInfoOperation.java

```diff
@@ -65,9 +65,10 @@
       ResponseHandler responseHandler, String blobIdStr, GetBlobOptionsInternal options,
       Callback<GetBlobResultInternal> callback, Time time) throws RouterException {
     super(routerConfig, routerMetrics, clusterMap, responseHandler, blobIdStr, options, callback, time);
-    operationTracker = new SimpleOperationTracker(routerConfig.routerDatacenterName, blobId.getPartition(),
+    operationTracker = new AdaptiveOperationTracker(routerConfig.routerDatacenterName, blobId.getPartition(),
         routerConfig.routerGetCrossDcEnabled, routerConfig.routerGetSuccessTarget,
-        routerConfig.routerGetRequestParallelism);
+        routerConfig.routerGetRequestParallelism, time, routerMetrics.getBlobInfoLocalColoLatency,
+        routerMetrics.getBlobInfoCrossColoLatency, routerConfig.routerLatencyToleranceQuantile);
```

Comment:
These the usual `Histogram` instances from codahle. I believe they account for all values since they were initiated (so since startup for us since this `Histogram` is created in `NonBlockingRouterMetrics`).